lib/saluki-event/src/eventd/mod.rs

```diff
@@ -1,9 +1,33 @@
-//! Event types
-mod priority;
-pub use self::priority::*;
+//! Events.
 
-mod alert;
-pub use self::alert::*;
+// TODO: Switch usages of `String` to `MetaString` since we should generally be able to intern these strings as they
+// originate in in the DogStatsD codec, where interning is already taking place.
+
+/// Alert type.
+#[derive(Clone, Copy, Debug)]
```

Comment:
While this was previously only `Clone`, enums are a prime candidate for being `Copy`. This means the value can be copied directly -- just copy the raw memory bytes and nothing else -- without any special logic.

Generally, integral types (signed and unsigned integers, floats, booleans) and "unit" enums (the variants don't have any fields) can stand to be `Copy` because their size in memory is no bigger than a _reference_ to this types.

This is to say that if you compare the size of this enum to a reference to this enum, the reference to it is much larger: 8 bytes versus 1 byte. References are a machine word (so 8 bytes on 64-bit architectures) while the compiler ends up bitpacking this enum in a single byte. The same thing holds if we use an integer type like `u32`: `&u32` is 8 bytes while `u32` itself is only 4 bytes.